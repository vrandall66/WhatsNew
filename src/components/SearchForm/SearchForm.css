.SearchForm {
  align-items: center;
  display: flex;
  justify-content: space-between;
  margin: 2%;
}

.SearchForm-input {
  background-color: #141527;
  border: none;
  color: #ffffff;
  font-size: 1.5em;
  height: 1.5em;
  width: 300px;
}

.SearchForm-button {
  background-color: #b569f1;
  border: none;
  border-radius: 75px;
  color: #ffffff;
  font-size: 1.5em;
  height: 55px;
  width: 225px;
}
