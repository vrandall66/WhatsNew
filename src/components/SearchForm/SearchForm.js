import React from 'react';
import './SearchForm.css';

const SearchForm = () => {
  return (
    <form>
      <input type='text' placeholder='Search for news articles here'/>
      <button type='button'>Search</button>
    </form>
  )
}

export default SearchForm;