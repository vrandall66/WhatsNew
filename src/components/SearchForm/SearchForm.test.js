import React from "react";
import { shallow } from "enzyme";
import SearchForm from "./SearchForm";

describe("SearchForm", () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<SearchForm />);
  });
  it("Should match the snapshot", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
