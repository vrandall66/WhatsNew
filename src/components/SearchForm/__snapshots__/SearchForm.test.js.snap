// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchForm Should match the snapshot 1`] = `
<form
  className="SearchForm"
>
  <input
    className="SearchForm-input"
    name="search"
    onChange={[Function]}
    placeholder="Search for news articles here"
    type="text"
    value=""
  />
  <button
    className="SearchForm-button"
    onClick={[Function]}
    type="button"
  >
    Search Now
  </button>
</form>
`;
